src/web/Text.tsx

```diff
@@ -15,6 +15,14 @@ import RX = require('../common/Interfaces');
 import Styles from './Styles';
 import Types = require('../common/Types');
 
+if (typeof document !== 'undefined') {
+    const textAsPseudoElement = '[data-text-as-pseudo-element]::before { content: attr(data-text-as-pseudo-element); }';
+    const style = document.createElement('style');
```

Comment:
This dynamic modification of the style is clever. I think we should use this same technique to eliminate the manual "box-sizing" override step that's currently needed for all ReactXP apps: https://microsoft.github.io/reactxp/docs/using-reactxp.html. If you feel like it, you're welcome to make this change as well -- and remove the step from the documentation.